config/environments/development.rb

```diff
@@ -37,5 +37,6 @@
 
   config.url_limit = 200
 
-  config.cmr_client_id = ENV['cmr_client_id'] || 'edsc-dev'
+  # This is the client ID that Earthdata Search sends to other services, i.e. CMR and OpenSearch
+  config.client_id = ENV['client_id'] || 'edsc-dev'
```

Comment:
Can we please change this back to just be cmr_client_id? The open search endpoint ultimately talks to the ECHO/CMR, which is why we specify it. I want to avoid confusion over the purpose of it and avoid having to change our operational environment variables twice as this change rolls out.